### PR TITLE
refactor: Added combo text component [191]

### PR DESCRIPTION
**Pull request by Astitva877**
## Description

Please do not leave this blank 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/lib/components/buttons/MethodButton.svelte [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
-    }
-    .api-type span{
-        font-weight: 500;
-    }
-</style>
\ No newline at end of file
+  .api-type {
+    height: 34px;
+    width: 56px;
+    border: 1px solid #313233;
```
Comment (parthdalaltech) [inline src/lib/components/buttons/MethodButton.svelte:23]: border-1 bootstrap class

File: src/lib/components/buttons/MethodButton.svelte [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
+    margin-right: 5px;
```
Comment (parthdalaltech) [inline src/lib/components/buttons/MethodButton.svelte:24]: mr-** bootstrap class

File: src/lib/components/buttons/MethodButton.svelte [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
+    font-size: 12px;
+  }
+  .api-type span {
+    font-weight: 500;
```
Comment (parthdalaltech) [inline src/lib/components/buttons/MethodButton.svelte:28]: font weight bootstrap class

File: src/lib/components/collections/req-res-section/sub-components/response-body-section/DefaultPage.svelte [hunk 4]
```diff
@@ -32,83 +33,50 @@
...
+        key={"Add Header"}
+        value={"alt + H"}
+        type="combo"
+      />
+      <ComboText
+        comboContainerClassProp={"d-flex align-items-center justify-content-between gap-5 mb-2"}
+        key={"Edit Body"}
+        value={"alt + B"}
+        type="combo"
+      />
```
Comment (parthdalaltech) [inline src/lib/components/collections/req-res-section/sub-components/response-body-section/DefaultPage.svelte:79]: create an object with keys comboContainerClassProp,key,value,type with their respective values and use map to iterate over the object and pass the values to combotext component

File: src/lib/components/text/ComboText.svelte [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      >
+    </div>
+  {/if}
+</div>
+
+<style>
+  .api-type {
+    height: 34px;
+    width: 56px;
+    border: 1px solid #313233;
```
Comment (parthdalaltech) [inline src/lib/components/text/ComboText.svelte:48]: border-1 bootstrap class

File: src/lib/components/text/ComboText.svelte [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    margin-right: 5px;
```
Comment (parthdalaltech) [inline src/lib/components/text/ComboText.svelte:49]: check margin class bootstrap

File: src/lib/components/text/ComboText.svelte [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  }
+  .api-type span {
+    font-weight: 500;
```
Comment (parthdalaltech) [inline src/lib/components/text/ComboText.svelte:52]: font weight class bootstrap

**Review by parthdalaltech — CHANGES_REQUESTED** [review]

File: src/lib/components/buttons/MethodButton.svelte (hunk 1 above)
Comment (Astitva877) [reply src/lib/components/buttons/MethodButton.svelte:23]: deprecated component.

File: src/lib/components/buttons/MethodButton.svelte (hunk 2 above)
Comment (Astitva877) [reply src/lib/components/buttons/MethodButton.svelte:24]: deprecated component.

File: src/lib/components/buttons/MethodButton.svelte (hunk 3 above)
Comment (Astitva877) [reply src/lib/components/buttons/MethodButton.svelte:28]: deprecated component.

File: src/lib/components/collections/req-res-section/sub-components/response-body-section/DefaultPage.svelte (hunk 4 above)
Comment (Astitva877) [reply src/lib/components/collections/req-res-section/sub-components/response-body-section/DefaultPage.svelte]: resolved

File: src/lib/components/text/ComboText.svelte (hunk 5 above)
Comment (Astitva877) [reply src/lib/components/text/ComboText.svelte]: resolved

File: src/lib/components/text/ComboText.svelte (hunk 6 above)
Comment (Astitva877) [reply src/lib/components/text/ComboText.svelte]: resolved

File: src/lib/components/text/ComboText.svelte (hunk 7 above)
Comment (Astitva877) [reply src/lib/components/text/ComboText.svelte]: resolved

**Review by parthdalaltech — APPROVED** [review]

**Review by parthdalaltech — APPROVED** [review]

**Review by parthdalaltech — APPROVED** [review]

**Pull request merged**